cli/report/convert.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package report
+
+import (
+	"bufio"
+	"encoding/json"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+// ReadFilesAndConcat reads json files in a directory that are one object per row,
+// and concats all objects into one single array
+func ReadFilesAndConcat(dir string) (results []interface{}, err error) {
```

Comment:
Note this approach isn't very efficient (needing to load all objects into an array *before* report generation). It's not a blocker for this PR, but in future iterations we're going to have to explore streaming input/output.

Also not sure why we need to export this function? Seems internal.

Reply:
This function is called by `convertAndGenerateTempAssetFile` in report.go hence exported.

Here are reasons to load all objects into array first: 

- The resultant file can be directly feed into [opa loader.All()](https://github.com/open-policy-agent/opa/blob/master/loader/loader.go#L82-L84), to avoid additional coding to implement similar function. 
- The resultant file is a human readable json file which is also being written to `output-path` as `raw_assets_<timestamp>.json`, I find useful for 1) ad-hoc inventory inspection, and 2) feeding into `opa` command directly for rego development and troubleshooting (Our policy-library test fixtures also have input asset files in one json array, while CAI exports are in the format of one json object per line)

Agreed that this approach is not ideal for performance. If we have a streaming solution, I would have the `report` command to use streaming approach and move the generation of one json array file to a separate feature.

Reply:
Alright, that sounds reasonable to me.